other/upgrade_2-1_postgresql.sql

```diff
@@ -2055,15 +2055,19 @@ CREATE INDEX {$db_prefix}ban_items_id_ban_ip ON {$db_prefix}ban_items (ip_low,ip
 --- helper function for ip convert
 /******************************************************************************/
 ---# the function migrate_inet
-CREATE OR REPLACE FUNCTION migrate_inet(val IN anyelement) RETURNS inet
-AS
-$$
-BEGIN
-   RETURN (trim(val))::inet;
-EXCEPTION
-   WHEN OTHERS THEN RETURN NULL;
-END;
-$$ LANGUAGE plpgsql;
+---{
+upgrade_query("
+	CREATE OR REPLACE FUNCTION migrate_inet(val IN anyelement) RETURNS inet
+	AS
+	$$
+	BEGIN
+	   RETURN (trim(val))::inet;
+	EXCEPTION
+	   WHEN OTHERS THEN RETURN NULL;
+	END;
+	$$ LANGUAGE plpgsql;"
+);
+---}
```

Comment:
still getting:

```
Error!
ERROR: unterminated dollar-quoted string at or near "$$ LANGUAGE plpgsql" LINE 1: $$ LANGUAGE plpgsql ^
```
and
```
Nov  4 21:20:47 cerebro postgres[79657]: [3-1] ERROR:  unterminated dollar-quoted string at or near "$$
Nov  4 21:20:47 cerebro postgres[79657]: [3-2]  BEGIN
Nov  4 21:20:47 cerebro postgres[79657]: [3-3]     RETURN (trim(val))::inet" at character 76
Nov  4 21:20:47 cerebro postgres[79657]: [3-4] STATEMENT:  CREATE OR REPLACE FUNCTION migrate_inet(val IN anyelement) RETURNS inet
Nov  4 21:20:47 cerebro postgres[79657]: [3-5]  AS
Nov  4 21:20:47 cerebro postgres[79657]: [3-6]  $$
Nov  4 21:20:47 cerebro postgres[79657]: [3-7]  BEGIN
Nov  4 21:20:47 cerebro postgres[79657]: [3-8]     RETURN (trim(val))::inet
Nov  4 21:20:47 cerebro postgres[79657]: [4-1] ERROR:  syntax error at or near "EXCEPTION" at character 1
Nov  4 21:20:47 cerebro postgres[79657]: [4-2] STATEMENT:  EXCEPTION
Nov  4 21:20:47 cerebro postgres[79657]: [4-3]     WHEN OTHERS THEN RETURN NULL
Nov  4 21:20:47 cerebro postgres[79657]: [5-1] ERROR:  unterminated dollar-quoted string at or near "$$ LANGUAGE plpgsql" at character 1
Nov  4 21:20:47 cerebro postgres[79657]: [5-2] STATEMENT:  $$ LANGUAGE plpgsql
```

Interestingly enough if I copy and paste the function creation directly to pgadmin, it works without fault